src/components/modal/tildel-veileder/tildel-veileder.tsx

```diff
@@ -100,6 +101,9 @@ function TildelVeilederRenderer({data, onSubmit, ident, onChange, btnOnClick}: T
 
 const mapStateToProps = (state, ownProps) => {
     const stateSlice = nameToStateSliceMap[ownProps.filtergruppe]
+        || (ownProps.filtergruppe === ListevisningType.enhetensOversikt
```

Comment:
I think this part is not necessary since function `nameToStateSliceMap` should do the same as: 
`(ownProps.filtergruppe === ListevisningType.enhetensOversikt ...`

Reply:
agreed